app/models/os_default_template.rb

```diff
@@ -3,7 +3,6 @@ class OsDefaultTemplate < ApplicationRecord
   belongs_to :template_kind
   belongs_to :operatingsystem
 
-  validates :provisioning_template_id, :presence => true
```

Comment:
Is this change intentional? Can you explain?

Reply:
I observed that the templates are mandatory in Hosts>OS>Templates because of this validation and hence, asterisks are shown. In order to make the template fields non-mandatory (i.e. removing asterisks), this change was done.

Reply:
I think we shouldn't remove validations, you can override the asterisk display by passing `:required => false` to the form helper